src/orders/views/OrderDetails/OrderNormalDetails/index.tsx

```diff
@@ -77,6 +89,25 @@ export const OrderNormalDetails: React.FC<OrderNormalDetailsProps> = ({
       first: 30
     }
   });
+
+  const {
+    data: customerAddresses,
+    loading: customerAddressesLoading
+  } = useCustomerAddressesQuery({
+    variables: {
+      id: order?.user?.id
+    },
+    skip: !order?.user?.id || !isAnyAddressEditModalOpen(params.action)
+  });
+  const handleCustomerChangeAddresses = async (
+    data: Partial<OrderCustomerAddressesEditDialogOutput>
+  ): Promise<any> => {
+    orderUpdate.mutate({
+      id,
+      input: data
+    });
+  };
```

Comment:
This async function's return type is marked as `Promise`, but it doesn't await nor return anything, so I guess it can be regular void function